From: =?UTF-8?q?Guido=20Berh=C3=B6rster?= <gber@opensuse.org>
Date: Sat, 19 Jan 2013 09:27:36 +0100
Subject: fix-double-escaping

Do not escape file names twice. Fixes an error when trying to create archives
with spaces.

Bug: http://bugs.debian.org/697493.
---
 src/main.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main.c b/src/main.c
index a13d737..a6a7f93 100644
--- a/src/main.c
+++ b/src/main.c
@@ -208,13 +208,13 @@ int main (int argc, char **argv)
 				g_free(_current_dir);
 				GSList *files = NULL;
 				_current_dir = g_path_get_basename(add_files);
-				files = g_slist_append(files,xa_escape_filename(_current_dir,"$'`\"\\!?* ()[]&|:;<>#"));
+				files = g_slist_append(files,g_strdup(_current_dir));
 				g_free(_current_dir);
 				g_free(add_files);
 				for (x = 1; x< argc; x++)
 				{
 					_current_dir = g_path_get_basename(argv[x]);
-					files = g_slist_append(files,xa_escape_filename(_current_dir,"$'`\"\\!?* ()[]&|:;<>#"));
+					files = g_slist_append(files,g_strdup(_current_dir));
 					g_free (_current_dir);
 				}
 				xa_execute_add_commands(archive,files,NULL);
